assets/schema_input.json

```
@@ -26,8 +38,18 @@
                 "exists": true,
                 "pattern": "^\\S+\\.f(ast)?q\\.gz$",
                 "errorMessage": "FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+            },
+            "rundir": {
+                "type": "string",
+                "format": "file-path",
```

This seems to fail on directories, or are we doing something wrong?

---

```suggestion
                "format": "directory-path",
```
This should fix it. I'm not familiar with this format.